src/test/java/org/jenkinsci/plugins/matrixauth/integrations/casc/ExportTest.java

```diff
@@ -0,0 +1,72 @@
+package org.jenkinsci.plugins.matrixauth.integrations.casc;
+
+import hudson.security.AuthorizationStrategy;
+import hudson.security.ProjectMatrixAuthorizationStrategy;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.Configurator;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import io.jenkins.plugins.casc.model.CNode;
+import io.jenkins.plugins.casc.model.Mapping;
+import org.jenkinsci.plugins.matrixauth.AuthorizationMatrixNodeProperty;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class ExportTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    @LocalData
```

Comment:
Hmm feels odd not to use JCasC to import the same input data.

Reply:
Going from non-CasC configuration to CasC export is a more expressive test case. Otherwise I need to "know" that CasC takes no shortcuts from between import and export (e.g. just echo back whatever it "imported" and never even applied to the running config).